src/services/ng2-uploader.ts

```diff
@@ -256,6 +256,15 @@ export class Ng2Uploader {
   }
 }
 
+export class UploadRejected{
+
+  public static get EXTENSION_NOT_ALLOWED():string { return "ExtensionNotAllowed"; }
```

Comment:
can you please use single quotes and be more strict about style. otherwise this looks okay.